src/main/kotlin/io/github/inductiveautomation/kindling/log/LogEvent.kt

```diff
@@ -3,31 +3,37 @@ package io.github.inductiveautomation.kindling.log
 import java.time.Instant
 
 sealed interface LogEvent {
+    var marked: Boolean
     val timestamp: Instant
```

Comment:
Can put `marked` at the end with a default value of false so you don't have to change all the constructor calls